src/model/database.py

```diff
@@ -0,0 +1,27 @@
+from model.table import *
+
+
+class Database:
+    def __init__(self, schema_data):
+        Database.__validate__(schema_data)
+        self.schema_data = schema_data
```

Comment:
databse.schema_data ?

Reply:
What is the meansing of the database ? 
database is a list of tables and database_name

`database.name`
`database.tables`
`database.schema_data` --->> ???